Sebastian-Sole/handicappin
# 🚚 chore: Clean components directory

## What was done
- Cleaned up the components directory by moving components into relevant subdirectories
- Fixed a bug in the hero where the homepage would crash if the user had played fewer than 10 rounds

## How to test
- Code review (for refactoring)
- Create a new user
- Log a round
- Be redirected to the homepage and see no error

## Future work
- None
